docs/contributing/style-guide.md

```diff
@@ -79,10 +79,10 @@ This gives a call to action for the reader or user to effectively get something
 By writing inclusively and using culturally neutral language, our words resonate with global audiences and make everyone feel welcome, no matter their race, gender, socioeconomic status, and ability.
 
 * Do not use idioms, slang, expressions, or terminology only understood by a specific region or group.
-* Avoid overly technical jargon.
+* Avoid overly-technical jargon.
```

Comment:
Hyphen isn't needed here. "Overly" is an adverb. See: 
* [Merriam Webster](https://www.merriam-webster.com/words-at-play/6-common-hypercorrections-and-how-to-avoid-them)
* [Cambridge Dictionary (U.S.)](https://dictionary.cambridge.org/us/dictionary/english/culturally)
* [Chicago Manual of Style](https://www.chicagomanualofstyle.org/qanda/data/faq/topics/HyphensEnDashesEmDashes/faq0093.html)


```suggestion
* Avoid overly technical jargon.
```